api/test_runs.go

```diff
@@ -52,6 +54,11 @@ func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
 		}
 		if pr != nil && aeAPI.IsFeatureEnabled("runsByPRNumber") {
 			filters.SHAs = getPRCommits(aeAPI, *pr)
+			if len(filters.SHAs) < 1 {
+				log.Warningf("PR %v returned no commits from GitHub", *pr)
+			} else {
+				log.Infof("PR %v returned %v commits: %s", *pr, len(filters.SHAs), strings.Join(filters.SHAs.SevenCharSHAs(), ","))
```

Comment:
Nit:
```suggestion
				log.Infof("PR %v returned %d commits: %s", *pr, len(filters.SHAs), strings.Join(filters.SHAs.SevenCharSHAs(), ","))
```

I'd like to avoid using `%v` so that `golint` can give me warnings if variables are not what I think they are...